tools/distgraphlaunch.py

```diff
@@ -0,0 +1,433 @@
+"""Launching tool for DGL distributed training"""
+import os
```

Comment:
why should we have separate executable scripts to launch partition jobs?
only one script is necessary.

Reply:
`distgraphlaunch.py` is a general script to launch any distributed job including training.  `dispatch_data.py` will run `distgraphlaunch.py` for data dispatching specifically.  So users only need to run `dispatch_data.py`.